lib/sidekiq/status.rb

```diff
@@ -0,0 +1,130 @@
+require 'sidekiq/api'
+
+module Sidekiq
+  class Status
```

Comment:
There is already a gem that uses the name "sidekiq-status" so I'd prefer another name.  Given that this is not an API designed for programmatic use, I think all this code should go back into `bin/sidekiqctl` and namespaced in `Sidekiqctl::Status`.